app/views/layouts/_header.html.erb

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-expand-lg bg-white">
+<nav class="<%= show_announcement %>">
```

Comment:
You can simply keep existing css classes, and add navbar class conditionally, using a helper makes this code harder to understand